src/Stepping.TmProviders.LocalTm.Core/Stepping/TmProviders/LocalTm/Steps/LocalTmStepConverter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Text;
+using Stepping.Core.Steps;
+
+namespace Stepping.TmProviders.LocalTm.Steps;
+
+public class LocalTmStepConverter : ILocalTmStepConverter
+{
+    protected IStepNameProvider StepNameProvider { get; }
+    protected IStepArgsSerializer StepArgsSerializer { get; }
+
+    public LocalTmStepConverter(IStepNameProvider stepNameProvider, IStepArgsSerializer stepArgsSerializer)
+    {
+        StepNameProvider = stepNameProvider;
+        StepArgsSerializer = stepArgsSerializer;
+    }
+
+    public virtual async Task<LocalTmStepModel> ConvertAsync(List<IStep> steps)
+    {
+        var result = new LocalTmStepModel();
+        foreach (var step in steps)
+        {
+            var stepType = step.GetType();
+            if (typeof(HttpRequestStep).IsAssignableFrom(stepType))
```

Comment:
```csharp
        foreach (var step in steps)
        {
            var stepType = step is HttpRequestStep ? typeof(HttpRequestStep) : step.GetType();

            result.Steps.Add(new LocalTmStepInfoModel(StepNameProvider.Get(stepType),
                await StepArgsSerializeAsync(step)));
        }
```

Reply:
Let's avoid converting steps using "if" or "switch". See [StepToDtmStepConvertResolver](https://github.com/TeamStepping/Stepping.NET/blob/60bab22347648b1d6f352c2bb1623707de00eb55/src/Stepping.TmProviders.Dtm.Grpc/Stepping/TmProviders/Dtm/Grpc/Steps/StepToDtmStepConvertResolver.cs).

Reply:
What if we don't convert the child classes to `HttpRequestStep`?

```csharp
// var stepType = step is HttpRequestStep ? typeof(HttpRequestStep) : step.GetType();
var stepType = step.GetType();
```

Reply:
`StepResolver` cannot resolve classes whose constructor is not `TArgs`, e.g. 
[RequestGitHubGetOrganizationStep](https://github.com/TeamStepping/Stepping.NET/blob/main/test/Stepping.TestBase/Fakes/RequestGitHubGetOrganizationStep.cs)

Reply:
I believe https://github.com/TeamStepping/Stepping.NET/pull/37 will resolve this problem.